### PR TITLE
CORE-209 feat: add complaint fields to voucher

### PR DESCRIPTION
**Pull request by s-eastwood**

### CODE REVIEW

File: src/common/Voucher.js [hunk 1]
```diff
@@ -15,6 +16,7 @@ function Voucher(id) {
   this.path = 'voucher';
   this.createPath = 'voucher';
   this.vouchersource = new VoucherSource(3); // TODO: 3 = 'manual', vouchersourceid field is not null!
+  this.complaint = new Complaint(3);
```
Comment (cdobson) [inline src/common/Voucher.js:19]: What's the 3 about here?

File: src/common/Voucher.js [hunk 2]
```diff
@@ -135,6 +138,7 @@ Voucher.prototype.toArray = function() {
       }
     }
     arr.vouchersourceid = this.vouchersource.id;
+    arr.complaintid = this.complaint.id;
```
Comment (cdobson) [inline src/common/Voucher.js:141]: Will this work when a voucher's created without a complaint?

Comment (cdobson) [reply src/common/Voucher.js:141]: Did you check this works @s-eastwood?

Comment (s-eastwood) [reply src/common/Voucher.js:141]: I can't remember, I'll double check now....but changes are needed for this once CORE-195 is done.

Comment (s-eastwood) [reply src/common/Voucher.js:144]: @cdobson I've made a change and believe this is working now

**Review by cdobson — APPROVED** [review]

**Pull request merged**